packages/wrangler/src/config/environment.ts

```diff
@@ -484,6 +484,15 @@ interface EnvironmentNonInheritable {
 		dataset?: string;
 	}[];
 
+	/**
+	 * A browser name that will be usable from the worker.
+	 */
+	browser:
+		| {
+				name: string;
```

Comment:
Does this correspond to the name of the browser instance, or to the name of the binding? If it's the name of the binding it should be called `binding`, with an additional property for `name` corresponding to the browser instance name/id

Reply:
This is just the binding name, we don't need any other. I'll change to `binding` then.

Reply:
Done.

Reply:
We don't have a precedent for bindings that have equivalent binding names and resource names. Using a pattern similar to [R2](https://developers.cloudflare.com/workers/wrangler/configuration/#r2-buckets) would be great here (`binding` and `browser_name`). That would allow (for instance) browser names that don't conform to javascript variable requirements

Reply:
For clarity, an API like:
```toml
browsers = [{ browser_name = "some browser name", binding = "binding_name" }]
```
would be more consistent with the rest of Wrangler's bindings

Reply:
I'm sorry, I don't think I understood the relation. In R2 you need to specify the bucket name you want to use, but in this binding there is no configuration parameter needed/expected, besides the name of the binding that will be exposed in the worker code. EWC is not expecting any additional parameters.